toolchest_client/files/tests/test_general.py

```diff
@@ -0,0 +1,50 @@
+import filecmp
+import os
+import pathlib
+
+import pytest
+
+from .. import assert_exists, check_file_size, files_in_path, sanity_check
+
+THIS_FILE_PATH = os.path.normpath(pathlib.Path(__file__).parent.resolve())
+
+
+def test_small_file():
+    small_file_path = f"{THIS_FILE_PATH}/data/very_small_file.txt"
+    with pytest.raises(ValueError):
+        sanity_check(small_file_path)
+
+def test_files_in_path():
+    tmp_dir = f"{THIS_FILE_PATH}/tmp"
+    tmp1 = f"{tmp_dir}/tmp1"
+    tmp2 = f"{tmp_dir}/tmp2"
+    sub_dir = f"{tmp_dir}/sub_dir"
+    tmp3 = f"{sub_dir}/tmp3"
+    file_paths = [tmp1, tmp2, tmp3]
+    for dir in [tmp_dir, sub_dir]:
+        if not os.path.exists(dir):
+            os.mkdir(dir)
```

Comment:
Could also: `os.makedirs(dir, exist_ok=True)`

Reply:
A follow-up to the try-catch-finally comment (since I can't reply to it directly, for some reason): looks like putting the `assert` in the `try` statement suppresses the `AssertionError` entirely, so I'm just going to leave it like this